src/collections/blog/2020-09-25-announcing-meshmates/index.mdx

```diff
@@ -0,0 +1,153 @@
+---
+title: "Announcing a new line of MeshMates"
+date: 2020-09-25 08:00:00 +0000
+author: our MeshMates with 🤍
+thumbnail: ./meshmate-announcement.png
+tags:
+ - Community
+ - Projects
+ - News
+published: true
+---
+
+import Ruth from "./meshmate-ruth-ikegah.png";
+import Kelechi from "./meshmate-kelechi-precious.png";
+import Tanuj from "./meshmate-tanuj-agarwal.png";
+
+import { BlogWrapper } from "../Blog.style.js";
+
+<BlogWrapper>
+
+  _[Layer5 MeshMates](/community/meshmates) are committed to helping community members be successful contributors. MeshMates aid in identifying areas of projects to engage within, working groups to join, and in helping community members grow in their open source and cloud native knowledge. By connecting one-on-one, MeshMates will share tips on how to have the best community experience possible._
+
+  <h4>
+    <a href="https://layer5.io/community/members/tanuj-agarwal">
```

Comment:
Change the link to the layer5-ng website's member page <Link to="/communtiy/members/tanuj-agarwal">.

Reply:
Rather change it to `<Link to="/community/members/tanuj-agarwal">`